generators/entity/index.js

```diff
@@ -183,8 +179,7 @@ module.exports = EntityGenerator.extend({
      * ask question for a field creation
      */
     _askForField : function(cb){
-        this.fieldId++;
-        this.log(chalk.green('\nGenerating field #' + this.fieldId + '\n'));
+        this.log(chalk.green('\nGenerating field\n'));
```

Comment:
I could readd it here again by checking the length of the relationship array but Im not sure how much that adds to the user experience. 
Instead we should show the list of the currently added fields as we do after adding a field.